spec/bolt/cli_spec.rb

```diff
@@ -775,14 +775,14 @@ def stub_config(file_content = {})
       end
 
       it "warns when using debug" do
-        expect(Bolt::Logger).to receive(:deprecation_warning)
-          .with(anything, /Command line option '--debug' is deprecated/)
+        expect(Bolt::Logger).to receive(:queue_message)
```

Comment:
This only verifies that the message gets queued, not printed. It's better to verify the behavior in this case:
```
expect(@log_output.readlines).to include(/Command line option '--debug' is deprecated/)
```